docs/Observable.md

```diff
@@ -0,0 +1,150 @@
+# Observable
+
+The `@dojo/shim/Observable` class is an implementation of the proposed [Observable specification](https://tc39.github.io/proposal-observable/).
+
+An `Observable` is used to simplify push-based data sources like events, timed events, or sockets.
+
+Here we create an Observable that publishes an event every 1 second:
+
+```typescript
+const timedObservable = new Observable(observer => {
```

Comment:
whitespace indentation on this example looks off, though that may be Github... should use tabs